src/phare/phare_init.py

```diff
@@ -1,41 +1,117 @@
 #!/usr/bin/env python3
 
+
 import pyphare.pharein
 from pyphare.pharein import Simulation
 from pyphare.pharein import MaxwellianFluidModel
 from pyphare.pharein import ElectromagDiagnostics
+from pyphare.pharein import ElectronModel
+import numpy as np
 
 # configure the simulation
 
 Simulation(
     smallest_patch_size=10,
-    largest_patch_size=64,
-    time_step_nbr=1000,        # number of time steps (not specified if time_step and final_time provided)
-    final_time=1.,             # simulation final time (not specified if time_step and time_step_nbr provided)
+    largest_patch_size=20,
+    time_step_nbr=5000,        # number of time steps (not specified if time_step and final_time provided)
+    final_time=5.,             # simulation final time (not specified if time_step and time_step_nbr provided)
     boundary_types="periodic", # boundary condition, string or tuple, length == len(cell) == len(dl)
-    cells=65,                  # integer or tuple length == dimension
-    dl=1./65,                  # mesh size of the root level, float or tuple
-    max_nbr_levels=2,          # (default=1) max nbr of levels in the AMR hierarchy
-    refinement_boxes = {"L0":{"B0":[(10,),(50,)]}},
-    diag_options = {"format":"phareh5", "options": {"dir": "phare_outputs"}}
+    cells=100,                # integer or tuple length == dimension
+    dl=0.3,                  # mesh size of the root level, float or tuple
+    max_nbr_levels=1,          # (default=1) max nbr of levels in the AMR hierarchy
+    #refinement_boxes={"L0": {"B0": [(10, ), (20, )]}},
```

Comment:
?

Reply:
indeed :-)
so, we go step by step. For now this PR is about having PHARE working with only a coarse level. I've tested 1 coarse level with a single or 4 patches with the Alfvén wave test as you've seen.

next step will be to have multiple (fixed) refinement level, but we need to adress #181  before.

that's why have not removed but commented the line. It's going to reappear in the next PR.

and the step after will consist in having a dynamic refinement criterion (or criteria) i.e. cell tagging.

Reply:
ok thanks!